src/components/numbered-code-snippet/numbered-code-snippet.js

```diff
@@ -50,7 +50,7 @@ export default class NumberedCodeSnippet extends React.PureComponent {
     injectedThemes.push(theme);
     const doc = getWindow().document;
     this.styleTag = doc.createElement('style');
-    this.styleTag.innerHTML = theme;
+    this.styleTag.innerHTML = `${theme}`;
```

Comment:
Curious about the reason for this rule. 🤓

Reply:
This rule is also from https://github.com/mapbox/eslint-config-mapbox/blob/master/xss.js 

https://github.com/Rantanen/eslint-plugin-xss/blob/master/docs/rules/no-mixed-html.md

If we were dealing with user input, we'd want to escape this value, but was able to wrap in a template literal instead.